commit 1dac62ee1a59c95ce069275c80d74fab559c80b1 (refs/changes/03/4103/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-06-01T10:23:07-07:00 (1 year, 4 months ago)
    
    TRITON-457 sdc-scripts update in TRITON-380 broke cns and sdc-manatee
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 6bfa4a2..deefaef 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 6bfa4a298d753685c6e083d10b70b5abdc39f66f
+Subproject commit deefaef587ed3bee2706cb6e53ee3468e682932e
